truss/remote/baseten/remote.py

```diff
@@ -205,6 +211,11 @@ def patch(
             return
         model_name = truss_handle.spec.config.model_name
         dev_version = get_dev_version(self._api, model_name)  # type: ignore
+        if not dev_version:
+            logger.error(
+                f"No development deployment found with model name: {model_name}"
```

Comment:
what does `logger.error` look like in the terminal? how does this compare to the ClickUsageError error messages?

Reply:
It just looks like a vanilla print (I have a screenshot in the PR description). Worth calling out that it doesn't stop execution of `truss watch`--e.g. a user could run `truss push` in a separate window to resolve this error and could continue using live reload without having to rerun `truss watch`. Do you think this is desirable, or is it too subtle?